modules/navier_stokes/include/auxkernels/INSFVMixingLengthTurbulentViscosityAux.h

```diff
@@ -0,0 +1,39 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "AuxKernel.h"
+
+// Computes the value of the eddy viscosity for the mixing length model.
+class INSFVVelocityVariable;
+
+class INSFVMixingLengthTurbulentViscosityAux : public AuxKernel
```

Comment:
this is the format for whole-class header dosctring
```suggestion
class INSFVVelocityVariable;
 
 /*
   * Computes the value of the eddy viscosity for the mixing length model.
   */ 
class INSFVMixingLengthTurbulentViscosityAux : public AuxKernel
```